corehq/apps/app_manager/suite_xml/sections/entries.py

```diff
@@ -10,14 +10,16 @@
 When forms work with multiple datums, they need to be named in a way that is predictable for app builders, who
 reference them inside forms. This is most relevant to the "select parent first" feature and to parent/child
 modules.  See ``update_refs`` and ``rename_other_id``, both inner functions in ``add_parent_datums``, plus
-`this comment <https://github.com/dimagi/commcare-hq/blob/c9fa01d1ccbb73d8f07fefbe56a0bbe1dbe231f8/corehq/apps/app_manager/suite_xml/sections/entries.py#L966-L971>`_
-on matching parent and child datums.
+`this comment`_ on matching parent and child datums.
 
+
+.. _this comment: https://github.com/dimagi/commcare-hq/blob/c9fa01d1ccbb73d8f07fefbe56a0bbe1dbe231f8/corehq/apps/app_manager/suite_xml/sections/entries.py#L966-L971  # noqa
 """
```

Comment:
Clever, though I came across this same thing (didn't want to put a python "comment" inside a multi-line string and determined that this does satisfy the linter:
```suggestion
.. _this comment: https://github.com/dimagi/commcare-hq/blob/c9fa01d1ccbb73d8f07fefbe56a0bbe1dbe231f8/corehq/apps/app_manager/suite_xml/sections/entries.py#L966-L971
"""  # noqa
```